Makefile

```diff
@@ -35,7 +35,7 @@ cover: devel-deps
 
 build: deps
 	mkdir -p build
-	for i in check-*; do \
+	for i in $(filter-out check-windows-%, $(wildcard check-*)); do \
```

Comment:
[comment][maybe okay]
Because `make build` accepts `TARGET_OSARCH` param, someone may think `make build TARGET_OSARCH="windows/amd64"` will build executables which run in Windows, but actually it doesn't build `check-windows-%`..